packages/syft/src/syft/store/db/stash.py

```diff
@@ -180,6 +180,54 @@ def _print_query(self, stmt: sa.sql.select) -> None:
     def unique_fields(self) -> list[str]:
         return getattr(self.object_type, "__attr_unique__", [])
 
+    @as_result(SyftException, StashException, NotFoundException)
+    @with_session
+    def get_by_unique_fields(
```

Comment:
Is this method used anywhere? I can't tell what it does.

Reply:
Not any longer, good catch @abyesilyurt. Thanks.

Reply:
Resolved in: [bc23abc](https://github.com/OpenMined/PySyft/pull/9298/commits/bc23abcb9eb47835459db9729e130a61347b27c9)